src/pudl/transform/classes.py

```diff
@@ -782,7 +852,14 @@ class AbstractTableTransformer(ABC):
     """Cached intermediate dataframes for use in development and debugging."""
```

Comment:
where the str comes from might be helpful to note here.

Reply:
Added a note pointing to the cache_dfs function.